src/app/api/common/delegates/getDelegates.ts

```diff
@@ -412,8 +424,8 @@ async function getDelegate(addressOrENSName: string): Promise<Delegate> {
     address,
     contracts.alligator?.address || "",
     slug,
-    contracts.governor.address.toLowerCase(),
-    contracts.token.address.toLowerCase()
+    contracts.governor.address,
+    contracts.token.address
   );
```

Comment:
Why are you removing the `toLowerCase`?

I think we have conflicting patterns between tenants here.